### ISSUE TITLE
New template: Preexisting data analysis

### ISSUE THREAD

**Issue by crsh** [crsh/prereg#12]
This is a template for aspredicted: https://psyarxiv.com/ph4q7/

I know there is also [another group](https://osf.io/cgw86/wiki/home/) working on something like this. We should reach out to them before starting on this.

**Comment by crsh** [comment]
- [x] Here's another one: https://psyarxiv.com/hvfmr

**Comment by henryy152** [comment]
> * [x]  Here's another one: https://psyarxiv.com/hvfmr

i think this template is already part of the package

**Comment by janinagrube** [comment]
The remaining templates that still need to be done each have their own new issue.
